package.json

```diff
@@ -8,6 +8,7 @@
     "scripts/serve-build.js"
   ],
   "dependencies": {
+    "@codemirror/lang-json": "^6.0.0",
```

Comment:
I think we would want to use monaco instead - We then get full openapi schema support, intellisense for javascript etc...